<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QDialog" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QTableView" name="tableDB">
         <property name="font">
          <font>
           <family>Century</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="6">
          <widget class="QLineEdit" name="lineEdit_count">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLineEdit" name="lineEdit_price">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEdit_fio">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="labelPrice">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Price</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="labelCount">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Quantity</string>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QLineEdit" name="lineEdit_date">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButtonDel">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>delete</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonAdd">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>add</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelFIO">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>FIO</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelID">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="labelType">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_Date">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Date</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEdit_id">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLineEdit" name="lineEdit_type">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButtonChange">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Change</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pushButton_reload">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>login</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>password</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>acces right</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="LabelAccessRight">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>a</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QTableView" name="tableView_pass"/>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="LabelPassword">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>l</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="LabelLogin">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>p</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_login">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>login</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="ref_pass">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="del_pass">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="add_pass">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_pass">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>password</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_status">
           <property name="font">
            <font>
             <family>Century</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>status</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
